src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1784,6 +1784,29 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("WIND_COV", 10.0f);
 		break;
 
+	case MAVLINK_MODE_EV:
+		configure_stream_local("ALTITUDE", 10.0f); // for avoidance
+		configure_stream_local("ATTITUDE", 25.0f);
+		configure_stream_local("ATTITUDE_TARGET", 10.0f);
+		configure_stream_local("CAMERA_TRIGGER", unlimited_rate); // for VIO
+		configure_stream_local("CAMERA_IMAGE_CAPTURED", unlimited_rate); //for VIO
+		configure_stream_local("DISTANCE_SENSOR", 10.0f);		// for avoidance
+		configure_stream_local("ESTIMATOR_STATUS", 1.0f);
+		configure_stream_local("EXTENDED_SYS_STATE", 1.0f);
+		configure_stream_local("GLOBAL_POSITION_INT", 10.0f);
+		configure_stream_local("GPS_RAW_INT", 1.0f);
+		configure_stream_local("HOME_POSITION", 0.5f);
+		configure_stream_local("HIGHRES_IMU", 250.0f);		// for VIO
```

Comment:
Set this to unlimited rate. The real rate is around 244 Hz anyway.

Reply:
From the tests with rovio, I'd rather keep it at a fixed rate of 250 Hz.

@bkueng @DanielePettenuzzo

Reply:
Can you explain the rationale? Mavlink scheduling will never let you hit 250 in reality. On top of that, the actual rate of sensor_combined is lower.

Reply:
> Set this to unlimited rate. The real rate is around 244 Hz anyway.

Where are you seeing this?

Reply:
I believe in `uorb top` messages per second.

Reply:
@mhkabir Open an issue if/when you see that with all the specifics. For the primary IMUs on nearly all of the current boards they should all be a 4 ms integration interval (250 Hz).

Reply:
Should the rate set to unlimited_rate or remain 250Hz?

Reply:
I would leave it unlimited_rate. Weren't going to have to change that assumed 250 Hz for some.